### PR TITLE
Create New page

### LINKED ISSUE #35: Page Creation
Add a button in the navigation menu that allow the user to create a new page. Since only the title is required, a new page with a default title ("Untitled Page") could be added. 

**Blocking**
- [x] Design

### PR DESCRIPTION
**Pull request by chrisdedman**
Added a button to create a new page on the sidebar.

### CODE REVIEW

File: src/app/(protected)/notebooks/[notebook_id]/[page_id]/page.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
 import { Loader } from "@/components/loader";
 import { useQuery, gql } from "@apollo/client";
+import _ from 'lodash';
 
 const NOTEBOOK_QUERY = gql`
   query GetNotebook($id: ID!) {
     notebook(id: $id) {
       id
       title
+      ownerId
```
Comment (PenguinBoi12) [inline src/app/(protected)/notebooks/[notebook_id]/[page_id]/page.tsx]: We don't need to query `ownerId` and `ownerType`


Part of the beauty of using GraphQL. We can query only the data we need.

File: src/app/(protected)/notebooks/[notebook_id]/page.tsx [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 import { Loader } from "@/components/loader";
 import { useQuery, gql } from "@apollo/client";
+import _ from 'lodash';
 
 const NOTEBOOK_QUERY = gql`
   query GetNotebook($id: ID!) {
     notebook(id: $id) {
       id
       title
+      ownerId
```
Comment (PenguinBoi12) [inline src/app/(protected)/notebooks/[notebook_id]/page.tsx]: We don't need to query `ownerId` and `ownerType`

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
image
This should show the "pointer" cursor. 

In general it seem good but there's still weird behaviour about the page list.
pages

**Comment by chrisdedman** [comment]
I fixed the issues based on your comments. However, I still had some trouble with the rendering of the array after clicking on the + button to create a new page. So, I used the ``location.reload();`` for now. I'm not sure why my array isn't rendered at the same time as I created the new page.

**Comment by chrisdedman** [comment]
> I fixed the issues based on your comments. However, I still had some trouble with the rendering of the array after clicking on the + button to create a new page. So, I used the `location.reload();` for now. I'm not sure why my array isn't rendered at the same time as I created the new page.

Fixed

**Review by PenguinBoi12 — APPROVED** [review]
This looks good! I think it's ready to be merged.

**Pull request merged**